src/components/Nav.js

```diff
@@ -22,7 +22,7 @@ if(KLAYTN_URL_NAME === 'Main Network'){
     { title: 'View Account Info', link: '/access', dropDown: false , icon: 'icon-info' },
     { title: 'Send KLAY & Token', link: '/transfer', search:'?next=transfer', dropDown: false ,  icon: 'icon-send' },
     { title: 'KLAY Faucet', link: '/faucet', dropDown: false ,search: '?next=faucet', icon: 'icon-faucet' },
-    { title: 'More', menuClass: 'menu-more',  link: null, isDropDown: true, icon: 'icon-more', dropDown: true , menu: [{name: 'Klaytnscope',pageMove: true, subLink: KLAYTN_SCOPE_URL}, {name: 'Klaytn IDE', pageMove: true, subLink: KLAYTN_IDE_URL },{name: 'Klaytn Docs', pageMove: true, subLink: KLAYTN_DOCS_URL }]},
+    { title: 'More', menuClass: 'menu-more',  link: null, isDropDown: true, icon: 'icon-more', dropDown: true , menu: [{name: 'Klaytnscope',pageMove: true, subLink: KLAYTN_SCOPE_URL}, {name: 'Klaytnfinder',pageMove: true, subLink: KLAYTN_FINDER_URL}, {name: 'Remix IDE', pageMove: true, subLink: KLAYTN_IDE_URL },{name: 'Klaytn Docs', pageMove: true, subLink: KLAYTN_DOCS_URL }]},
```

Comment:
```suggestion
    { title: 'More', menuClass: 'menu-more',  link: null, isDropDown: true, icon: 'icon-more', dropDown: true , menu: [{name: 'Klaytnfinder',pageMove: true, subLink: KLAYTN_FINDER_URL}, {name: 'Klaytnscope',pageMove: true, subLink: KLAYTN_SCOPE_URL}, {name: 'Remix IDE', pageMove: true, subLink: KLAYTN_IDE_URL },{name: 'Klaytn Docs', pageMove: true, subLink: KLAYTN_DOCS_URL }]},
```

Reply:
@praveen-klaytn can we make klaytnfinder before klaytnscope?

Reply:
Hi @kjhman21 , I have changed the order as requested.   
  
And I think, we need to make sure **KLAYTN\_FINDER\_URL** variable in added in **environment variables list of CI/CD** with [https://klaytnfinder.io/](https://www.klaytnfinder.io/) value for production and [https://baobab.klaytnfinder.io/](https://baobab.klaytnfinder.io/) value for baobab network. Default value of **KLAYTN\_FINDER\_URL** is  [https://klaytnfinder.io/](https://www.klaytnfinder.io/)

Reply:
@praveen-klaytn Seems okay, but I will double-check after version update.